lib/measures/tbd/resources/psi.rb

```diff
@@ -1391,13 +1387,43 @@ def processTBD(os_model, psi_set, io_path = nil, schema_path = nil)
   # 4. array of linked outside- or ground-facing surfaces
   edges.values.each do |e|
     next unless e.has_key?(:psi)
-    p = e[:psi].values.max
-    next unless p > 0.000
-    t = e[:psi].key(p)
+    next unless e.has_key?(:set)
+    v = e[:psi].values.max
+    p = e[:set]
+    next unless v > 0.000
+    t = e[:psi].key(v)
     l = e[:length]
-    edge = { psi: p, type: t, length: l, surfaces: e[:surfaces].keys }
-    io[:edges] << edge
+    edge = { psi: p, type: t, length: l, surfaces: e[:surfaces].keys, count: 1 }
+
```

Comment:
A count (e.g. 1, 2, 3, ...) is indeed required for multiple edges (e.g. 4 sides of a skylight)